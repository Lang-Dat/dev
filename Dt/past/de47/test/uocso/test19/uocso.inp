100
186737633 745605796
48868306 201779112
624415187 939270533
795670047 147623980
347048263 134737512
443095457 468588612
553760555 11603320
786714017 60217144
250168345 634999631
134342149 309602772
654859797 602572340
95908402 568727415
270610678 68054668
963671027 683853890
483463668 800494353
480938946 720356635
945938444 442498215
39483791 717020899
767527671 817004310
908007475 77342600
518729911 828767927
410782593 923772940
470865730 100331807
598657069 544846839
450702022 655332006
723779624 728309777
287151826 829956534
336083281 137468368
245793573 786754267
592300538 307651146
106918957 369196120
784001794 995427439
163159974 211961607
862286603 251260561
641840708 506967933
803163051 858746687
156724312 231711474
679176819 754105727
496988430 179298376
893599524 386805580
380943983 640837485
946810322 531316976
446117186 843356271
748055812 85101501
185227370 555744985
513626067 120608426
119888134 475874192
170247580 701386469
826447915 731843426
951841926 123957772
752353733 548524701
876415339 335289284
475386781 306349718
43275227 462143938
685259088 993774813
853463094 316890673
804121288 63243173
530882972 522994296
751569288 911423078
772392644 141560519
719804717 206943013
825239453 362613201
292485327 650182863
622833545 298052133
280396515 974019853
446316032 132929014
422928906 18950792
677087917 605843722
880236572 969581893
73552735 10377678
878762418 54520770
529062715 609612560
469620835 162038870
272887491 494331842
861973533 482126728
492378769 53975747
26887712 274847518
550423262 425740965
973094343 41257894
194691168 353367870
993186115 864046319
965068689 270314041
601131419 675500890
414697857 843400160
875501337 427690640
476271827 710254373
225586173 778544263
524469284 518523988
904595686 939702740
57293911 749931758
86204561 335622866
177536343 871226284
612472054 148853409
900570070 149624581
548135405 48426891
691675822 680274551
956169602 615780688
885453644 546959484
140020557 138973495
636767950 614204055
