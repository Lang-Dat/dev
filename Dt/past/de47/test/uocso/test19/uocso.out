192
320
32
960
1024
768
96
64
384
144
48
192
256
128
120
128
384
8
96
320
32
384
64
32
192
256
768
20
96
192
64
128
640
16
48
48
80
80
80
960
144
48
64
192
96
24
96
96
64
768
24
144
32
8
240
64
96
384
160
48
16
32
48
16
64
88
160
72
384
128
288
480
192
32
192
64
112
128
256
1120
64
64
128
768
320
32
32
160
128
64
48
192
128
256
256
64
96
240
64
768
